web/client/plugins/userExtensions/ExtensionsPanel.jsx

```diff
@@ -96,9 +99,20 @@ const ExtensionsPanel = ({
             onSelect={onSelect} />
     </BorderLayout>);
 };
-export default connect(
-    createSelector(userPluginsSelector, extensions => ({extensions})),
-    {
-        onSelect: (extension) => updateUserPlugin(extension.name, { active: !extension.active })
-    }
+
+
+export default compose(
+    connect(
+        createSelector(userPluginsSelector, extensions => ({extensions})),
+        {
+            onSelect: (extension) => updateUserPlugin(extension.name, { active: !extension.active })
+        }
+    ),
+    // get plugins configuration
+    withPluginsDefinition(),
+    // localize extensions labels
+    localizedProps('extensions', 'title'),
+    localizedProps('extensions', 'description'),
+    // add filter state handling
+    withState('filterText', 'onFilter')
```

Comment:
Why this change? Wasn't the hook good enough?

Reply:
It was trowing errors due to wrong usage of hooks (you're violating hooks rules") that I couldn't understand. So I came back to a well known solution.